packages/https-proxy/test/integration/proxy_spec.coffee

```diff
@@ -152,6 +154,37 @@ describe "Proxy", ->
           proxy: "http://localhost:3333"
         })
 
+    ## https://github.com/cypress-io/cypress/issues/771
+    it "generates certs and can proxy requests for HTTPS requests to IPs", ->
+      @sandbox.spy(@proxy, "_generateMissingCertificates")
+      @sandbox.spy(@proxy, "_getServerPortForIp")
+
+      remove = (folder, file) =>
+        fs.removeAsync(path.join(folder, file))
```

Comment:
there may already be a method to clear all the certs - if not we should probably create one